src/api/zoops.ts

```diff
@@ -1,21 +1,61 @@
-import express, {Request, Response, NextFunction} from "express";
+import express, { Request, Response, NextFunction } from "express";
 import { PrismaClient } from "@prisma/client";
 import requireUser from "../utils/requireUser";
 
 const prisma = new PrismaClient();
 
 const zoopsRouter = express.Router();
 
-
 // GET /api/zoops
+// TODO: Remove Unnecessary feilds from selections? 
 zoopsRouter.get("/", async (req, res, next): Promise<void> => {
   try {
-    const zoops = await prisma.zoop.findMany();
-    res.send({zoops});
+    const zoops = await prisma.zoop.findMany({
+      include: {
+        faves: {
+          include: {
+            faver: {
+              select: {
+                id: true,
+                email: true,
+                dateCreated: true,
+                username: true,
+                zoopsWritten: true,
+                zoopsReceived: true,
+                faves: true,
+              },
+            },
```

Comment:
Would be nice to do this programmatically - maybe by mapping/reducing over the list of keys on the User object and then setting password to false? I don't know... it just doesn't sit well with me to have to write out every field name just so we can avoid password - what if a new field is added to User later?

Reply:
I imported a package to handle this. I wasn't sure how to go about handling it on my own.